Marlin/planner.cpp

```diff
@@ -1128,13 +1128,19 @@ void Planner::_buffer_steps(const int32_t (&target)[XYZE], float fr_mm_s, const
     }
   #endif // XY_FREQUENCY_LIMIT
 
+  block->nominal_speed = max_stepper_speed; // (mm/sec) Always > 0
+  block->nominal_rate = CEIL(block->step_event_count * inverse_mm_s);  // (step/sec) Always > 0
```

Comment:
`inverse_mm_s` is now called `inverse_secs` because that's its actual meaning. Are you working from the latest code?

Reply:
bugfix-1.1.x?